zerver/lib/export.py

```diff
@@ -352,13 +352,15 @@ def sanity_check_output(data: TableData) -> None:
 
 
 def write_data_to_file(output_file: Path, data: Any) -> None:
+    logging.info("Writing to %s", output_file)
     with open(output_file, "wb") as f:
         # Because we don't pass a default handler, OPT_PASSTHROUGH_DATETIME
         # actually causes orjson to raise a TypeError on datetime objects. This
         # is what we want, because it helps us check that we correctly
         # post-processed them to serialize to UNIX timestamps rather than ISO
         # 8601 strings for historical reasons.
         f.write(orjson.dumps(data, option=orjson.OPT_INDENT_2 | orjson.OPT_PASSTHROUGH_DATETIME))
+    logging.info(" (done writing)")
```

Comment:
I image it might be nicer to have this by just "Finished writing %s", but would need to look at output to have a strong opinion.

Reply:
After looking at the output, I think probably it is important to say what we finished, because otherwise the parallel message.partial.json => message.json phase will appear to be a randomly sorted collecting to start/finish lines that are hard to interpret.

```
2021-12-08 22:40:58.444 INFO [] Launching 6 PARALLEL subprocesses to export UserMessage rows
2021-12-08 22:41:00.802 INFO [] Starting UserMessage batch thread 3
2021-12-08 22:41:00.803 INFO [] Thread 3 processing /tmp/zulip-export-cfmdwaoa/messages-000001.json
2021-12-08 22:41:00.811 INFO [] Starting UserMessage batch thread 2
2021-12-08 22:41:00.842 INFO [] Fetched UserMessages for /tmp/zulip-export-cfmdwaoa/messages-000001.json
2021-12-08 22:41:00.842 INFO [] Writing to /tmp/zulip-export-cfmdwaoa/messages-000001.json
2021-12-08 22:41:00.843 INFO []  (done writing)
2021-12-08 22:41:00.845 INFO [] Starting UserMessage batch thread 0
2021-12-08 22:41:00.866 INFO [] Starting UserMessage batch thread 5
2021-12-08 22:41:00.930 INFO [] Starting UserMessage batch thread 1
2021-12-08 22:41:00.933 INFO [] Starting UserMessage batch thread 4
```

Reply:
that would feel spammier to me and harder to scan

Reply:
Maybe it'd be better to only print a line when it succeeds writing a file?  I imagine serialization bugs are rare, and then we only print a single atomic line per file.

Reply:
I guess writing a line after success is the design we had before this PR, probably for that reason.  So yeah, my proposal is to still do this cleanup of moving those log lines into the helper function that writes the files, but have it just display at the end of attempting to write a file, rather than 2 lines per entry.